File: DSA/Python/Array/reverseArr.py
arr = [12,4,5,6,7,8,9,3,4,3]
print(len(arr)/2)

for i in range(len(arr)//2):
    arr[i], arr[len(arr)-1-i] = arr[len(arr)-1-i], arr[i]

print(arr)

File: DSA/JS/Array/ReverseArr.js

const arr = [1, 2, 3, 4, 5, 8, 9, 1, 23];

for (let i=0; i<arr.length/2; i++){
    console.log(arr.length/2);
    [arr[i], arr[arr.length-1-i]] = [arr[arr.length-1-i], arr[i]]
}

console.log(arr);

console.log(arr.reverse());

